make-rules/shared-macros.mk

```diff
@@ -728,6 +746,7 @@ IPS2TGZ = 	$(WS_TOOLS)/ips2tgz
 
 INS.dir=        $(INSTALL) -d $@
 INS.file=       $(INSTALL) -m 444 $< $(@D)
+INS.exec=       $(INSTALL) -m 655 $< $(@D)
```

Comment:
Nowhere - just found the lines while looking for "install" usecases and adding this seemed to be right for completeness...

Indeed, now I see there are very few hits of original `INS.dir` and `INS.file` in the existing sources... But the idea seems reasonable to hide the CLI arguments of a particular `$(INSTALL)`'s implementation to get a job done.